src/utils/components/DiskModal/DiskFormFields/StorageClass/AlertedStorageClassSelect.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC, useState } from 'react';
+
+import DefaultStorageClassAlert from './DefaultStorageClassAlert';
+import StorageClassSelect from './StorageClassSelect';
+
+export type AlertedStorageClassSelectProps = {
+  storageClass: string;
+  setStorageClassName: (scName: string) => void;
```

Comment:
i think you can use React actions for the type, there are some samples in our code base

Reply:
you mean Dispatch<SetStateAction<string>>?

Reply:
yes

Reply:
changed